keyboards/dp60/keymaps/via/keymap.c

```diff
@@ -1,31 +1,31 @@
-      _______,        _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
-      _______,_______,_______,                        _______,                        _______,_______,_______,_______),
+    [1] = LAYOUT_all(
+      KC_GRV, KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,KC_TRNS,KC_DEL,
+      RESET,   KC_TRNS,KC_UP,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_PAUS,KC_SLCK,KC_PSCR,KC_PGUP,KC_TRNS,KC_INS,
+      KC_TRNS,        KC_LEFT,KC_DOWN,KC_RIGHT,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,KC_TRNS,KC_HOME,KC_END,KC_TRNS, KC_TRNS,
+      KC_TRNS,KC_TRNS,        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_PGDN,KC_TRNS,KC_TRNS,
+      KC_TRNS,KC_TRNS,KC_TRNS,                        KC_TRNS,                        KC_TRNS,MO(2),TG(0),KC_TRNS),
 
-    [3] = LAYOUT_60_ansi_split_bs_rshift(
-      _______, _______,_______,  _______,_______,  _______,_______,  _______,_______,  _______,_______, _______,_______,_______,_______,
-      _______,   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
-      _______,        _______,_______,_______,_______,_______,_______,_______, _______,_______,_______,_______,_______,
-      _______,        _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
-      _______,_______,_______,                        _______,                        _______,_______,_______,_______)    
-};
\ No newline at end of file
+    [2] = LAYOUT_all(
+      KC_TRNS, KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS,   KC_TRNS,KC_VOLU,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_MUTE,        KC_MPRV,KC_VOLD,KC_MNXT,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS,       RGB_TOG,RGB_MOD,RGB_RMOD,RGB_VAI,RGB_VAD,RGB_HUI,RGB_HUD,RGB_SAI,RGB_SAD,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS,KC_TRNS,KC_TRNS,                        KC_MPLY,                        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS),
+
+    [3] = LAYOUT_all(
+      KC_TRNS, KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS,   KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS,        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS, KC_TRNS,       KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+      KC_TRNS,KC_TRNS,KC_TRNS,                        KC_TRNS,                        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS)
```

Comment:
```suggestion
      _______, _______,_______,  _______,_______,  _______,_______,  _______,_______,  _______,_______, _______,_______,_______,_______,
      _______,   _______,KC_VOLU,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
      KC_MUTE,        KC_MPRV,KC_VOLD,KC_MNXT,_______,_______,_______,_______, _______,_______,_______,_______,_______, _______,
      _______, _______,       RGB_TOG,RGB_MOD,RGB_RMOD,RGB_VAI,RGB_VAD,RGB_HUI,RGB_HUD,RGB_SAI,RGB_SAD,_______,_______,_______,
      _______,_______,_______,                        KC_MPLY,                        _______,_______,_______,_______),

    [3] = LAYOUT_all(
      _______, _______,_______,  _______,_______,  _______,_______,  _______,_______,  _______,_______, _______,_______,_______,_______,
      _______,   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
      _______,        _______,_______,_______,_______,_______,_______,_______, _______,_______,_______,_______,_______,_______,
      _______, _______,       _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
      _______,_______,_______,                        _______,                        _______,_______,_______,_______)
```

Reply:
@fauxpark Thanks for suggestions. Isn't KC_TRNS reccomended instead of _______ inside via's keymap.c?

Reply:
It's exactly the same, `_______` just makes the keymap more readable.

Reply:
OK, awesome. Will commit now.